org.jacktrip.JackTrip.yml

```
@@ -0,0 +1,50 @@
+app-id: org.jacktrip.JackTrip
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: jacktrip
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  # - --socket=wayland
+  # Needs network access
+  - --share=network
+  # Pipewire/Jack
+  - --filesystem=xdg-run/pipewire-0
+  # For setting realtime priority for network thread?
+  - --socket=system-bus
```

```suggestion
  - --system-talk-name=org.freedesktop.RealtimeKit1
```

But it doesn't work (ie it doesn't set the real time priority it's a sandboxing limitation)

---

AFAIK RealtimeKit doesn't even work for JACK or Pipewire.

---

if it's a different dbus-name then use the right one. But `--socket=system-bus` is a no no in general.

---

I removed it entirely.